### DISCUSSION TITLE
"Busy" status - allow to set autoclearing after a manually specified date

### DISCUSSION THREAD

**Discussion by JanChec** [community/community#17853] [Other Feature Feedback, Questions, & Ideas]
Hey! :slightly_smiling_face: 

We're using [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and automatic code reviewer assignment. This helps a lot because we get automatically requested to take a look at our owned files and we don't have to have some custom solution to rotate over code reviewers from the owners group - it's built-in now. However, if someone is on vacation, to prevent assigning them they have to set themselves as "Busy". It would be best to have it somehow done automatically to avoid human errors, but it seems that it would require somehow passing the schedule to GitHub, which might be an overkill.

Still, one thing may be significantly improved - **allowing to set any date in the "Clear status:" input**. So that when someone sets it for their vacations, they can at the same time specify when exactly they come back. Then the status will be reset automatically. Without it people will have to either set one of the options or not set autoclearing at all, possibly forgetting about it and not receiving code reviews for an indefinite time... This is a worse scenario than forgetting about setting "Busy" at all and having PR owners to pick manually someone else for CR.

Thanks in advance for looking into it! :slightly_smiling_face:

**Comment by siminapasat** [comment]
Hi @Kern3l ! That's a great suggestion. Thank you, noted it down.
